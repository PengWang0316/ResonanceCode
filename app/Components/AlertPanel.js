import React from 'react';

import styles from '../styles/AlertPanel.module.css';

const AlertPanel = ({
  id = '', type = 'success', children = '', style = {}
}) => (
  <div id={id} className={`alert alert-${type} mt-3 ${styles.alertPanel}`} style={style} role="alert">
    {children}
  </div>
);
export default AlertPanel;
