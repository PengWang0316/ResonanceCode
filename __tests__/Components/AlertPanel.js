import React from 'react';
import renderer from 'react-test-renderer';

import AlertPanel from '../../app/Components/AlertPanel';

describe('AlertPanel Test', () => {
  test('AlertPanel snapshot', () => expect(renderer.create(<AlertPanel />).toJSON()).toMatchSnapshot());
});
