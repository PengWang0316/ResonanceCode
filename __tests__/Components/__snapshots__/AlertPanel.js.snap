// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertPanel Test AlertPanel snapshot 1`] = `
<div
  className="alert alert-success mt-3 text-center alertPanel"
  id=""
  role="alert"
  style={Object {}}
>
  
</div>
`;
